components/Python/fileConnectors/s3_file_source/component.json

```diff
@@ -0,0 +1,92 @@
+{
+  "engineType": "Python",
+  "userStandalone": false,
+  "language": "Python",
+  "name": "s3_file_source",
+  "label": "S3 File Source",
+  "description": "Import a bucket/key from S3 to a file",
+  "program": "main.py",
+  "componentClass": "S3FileSource",
+  "group": "Connectors",
+  "useMLOps": true,
+  "inputInfo": [],
+  "outputInfo": [
+    {
+        "description": "File name",
+        "label": "File-Name",
+        "defaultComponent": "",
+        "type": "str",
+        "group": "data"
+    }
+  ],
+  "arguments": [
+    {
+      "key": "aws_access_key_id",
+      "label": "AWS Access Key",
```

Comment:
again capital letters

Reply:
Thanks for bring this up..
Sticking to capitalization of words of significance and lowercase for all others